drakrun/drakrun/lib/sample_startup.py

```diff
@@ -64,23 +91,25 @@ def get_dll_startup_command(pe_data: bytes) -> str:
             for e in pe.DIRECTORY_ENTRY_EXPORT.symbols
         ]
     except AttributeError:
-        return "regsvr32 /s %f"
+        return "DllMain"
```

Comment:
I don't know why, but in the old code it ended called regsvr32 by default, right?